### PR TITLE
add stricter tests for #251; styling

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
I have added a few more strict tests for the multiple CIs functionality. I am skipping them on `CRAN` just to make our lives a bit easier in case we want to make any further changes to these outputs.

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
@strengejacke The only outstanding issue that remains to be addressed now is getting rid of these `warnings`:

``` r
set.seed(123)
# fake a 2nd response variable
npk2 <- within(npk, foo <- rnorm(24))

# model
m <- manova(cbind(yield, foo) ~ block + N * P * K, npk2)

# parameters
as.data.frame(parameters::model_parameters(m,
  ci = 0.99,
  omega_squared = "partial"
))
#> Warning in params[[es]][valid_rows] <- fx[[es]]: number of items to replace is
#> not a multiple of replacement length
#> Warning in params[[ci_low]][valid_rows] <- fx$CI_low: number of items to replace
#> is not a multiple of replacement length
#> Warning in params[[ci_high]][valid_rows] <- fx$CI_high: number of items to
#> replace is not a multiple of replacement length
#>   Parameter      Pillai df          F           p Omega2_partial Omega2_CI_low
#> 1     block 0.883067423  5 1.89748411 0.096377338   -0.087898842             0
#> 2         N 0.607644076  1 8.51788443 0.005824266    0.006461038             0
#> 3         P 0.066145232  1 0.38956676 0.686335357   -0.040693350             0
#> 4         K 0.387877220  1 3.48512550 0.067237566   -0.041330147             0
#> 5       N:P 0.105732911  1 0.65028783 0.540841322   -0.040917194             0
#> 6       N:K 0.173682170  1 1.15603452 0.350194388   -0.043402707             0
#> 7       P:K 0.003191087  1 0.01760717 0.982574568   -0.043467796             0
#> 8 Residuals          NA 12         NA          NA             NA            NA
#>   Omega2_CI_high
#> 1      0.0000000
#> 2      0.3919812
#> 3      0.0000000
#> 4      0.0000000
#> 5      0.0000000
#> 6      0.0000000
#> 7      0.0000000
#> 8             NA
```

<sup>Created on 2020-11-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9001)</sup>

**Comment by strengejacke** [comment]
It's no warning, it doesn't work correctly. @mattansb do you have an idea what _effectsize_ returns for `manova()`? obviously, `model_parameters()` and `eta_squared()` refer to different parameters, whereas `summary()` is the same as `model_parameters()`.

``` r
library(easystats)
#> # Attaching packages
set.seed(123)
# fake a 2nd response variable
npk2 <- within(npk, foo <- rnorm(24))

# model
m <- manova(cbind(yield, foo) ~ block + N * P * K, npk2)

summary(m)
#>           Df  Pillai approx F num Df den Df   Pr(>F)   
#> block      5 0.88307   1.8975     10     24 0.096377 . 
#> N          1 0.60764   8.5179      2     11 0.005824 **
#> P          1 0.06615   0.3896      2     11 0.686335   
#> K          1 0.38788   3.4851      2     11 0.067238 . 
#> N:P        1 0.10573   0.6503      2     11 0.540841   
#> N:K        1 0.17368   1.1560      2     11 0.350194   
#> P:K        1 0.00319   0.0176      2     11 0.982575   
#> Residuals 12                                           
#> ---
#> Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

model_parameters(m)
#> Parameter |   Pillai | df |    F |     p
#> ----------------------------------------
#> block     |     0.88 |  5 | 1.90 | 0.096
#> N         |     0.61 |  1 | 8.52 | 0.006
#> P         |     0.07 |  1 | 0.39 | 0.686
#> K         |     0.39 |  1 | 3.49 | 0.067
#> N:P       |     0.11 |  1 | 0.65 | 0.541
#> N:K       |     0.17 |  1 | 1.16 | 0.350
#> P:K       | 3.19e-03 |  1 | 0.02 | 0.983
#> Residuals |          | 12 |      |

eta_squared(m)
#> Response | Parameter | Eta2 (partial) |       90% CI
#> ----------------------------------------------------
#> foo      |     block |           0.20 | [0.00, 0.35]
#> foo      |         N |           0.09 | [0.00, 0.38]
#> foo      |         P |       5.10e-03 | [0.00, 0.19]
#> foo      |         K |       3.94e-03 | [0.00, 0.17]
#> foo      |       N:P |       4.69e-03 | [0.00, 0.18]
#> foo      |       N:K |       1.39e-04 | [0.00, 0.00]
#> foo      |       P:K |       1.92e-05 | [0.00, 0.00]
#> yield    |     block |           0.65 | [0.16, 0.82]
#> yield    |         N |           0.51 | [0.15, 0.71]
#> yield    |         P |           0.04 | [0.00, 0.32]
#> yield    |         K |           0.34 | [0.03, 0.60]
#> yield    |       N:P |           0.10 | [0.00, 0.40]
#> yield    |       N:K |           0.15 | [0.00, 0.45]
#> yield    |       P:K |       2.59e-03 | [0.00, 0.15]
```

<sup>Created on 2020-11-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

File: R/robust_estimation.R [hunk 1]
```diff
@@ -40,7 +40,9 @@
 #'     vcov_type = "CR2",
 #'     vcov_args = list(cluster = iris$cluster)
 #'   )
-#' }}
+#' }
+#' }
+#'
```
Comment (strengejacke) [inline R/robust_estimation.R:45]: I don't like this styling rule, as it adds unnecessary empty lines to the examples in help-pages.

File: R/select_parameters.R [hunk 2]
```diff
@@ -35,23 +35,28 @@
 #'     data = iris
 #'   )
 #'   select_parameters(model)
-#' }}
+#' }
+#' }
```
Comment (strengejacke) [inline R/select_parameters.R:39]: Same here...

**Review by strengejacke — CHANGES_REQUESTED** [review]

File: tests/testthat/test-model_parameters.aov_es_ci.R [hunk 3]
```diff
@@ -102,9 +111,488 @@ if (require("insight") && require("effectsize") && require("testthat") && requir
...
+    expect_equal(colnames(mp), c(
+      "Response", "Parameter", "Sum_Squares", "df", "Mean_Square", "F", "p",
+      "Omega2_partial", "Omega2_CI_low", "Omega2_CI_high", "Eta2_partial",
+      "Eta2_CI_low", "Eta2_CI_high", "Epsilon2_partial", "Epsilon2_CI_low",
+      "Epsilon2_CI_high"
+    ))
+  })
+}
+
+# stricter tests ---------------------------------------------------------
```
Comment (strengejacke) [inline tests/testthat/test-model_parameters.aov_es_ci.R:123]: I think you need to add the `tolerance` argument, so tests don't fail on different systems. Furthermore, there's the new `expect_equivalent()`, which is similar to `expect_equal()`, but ignores attributes in objects.

**Comment by mattansb** [comment]
Sorry, there just wasn't a method for `manova`, so it was reverting to the `maov` method.
Should be fixed now (on dev).

``` r
library(effectsize)
library(parameters)

set.seed(123)
# fake a 2nd response variable
npk2 <- within(npk, foo <- rnorm(24))

# model
m <- manova(cbind(yield, foo) ~ block + N * P * K, npk2)


# manova
summary(m)
#>           Df  Pillai approx F num Df den Df   Pr(>F)   
#> block      5 0.88307   1.8975     10     24 0.096377 . 
#> N          1 0.60764   8.5179      2     11 0.005824 **
#> P          1 0.06615   0.3896      2     11 0.686335   
#> K          1 0.38788   3.4851      2     11 0.067238 . 
#> N:P        1 0.10573   0.6503      2     11 0.540841   
#> N:K        1 0.17368   1.1560      2     11 0.350194   
#> P:K        1 0.00319   0.0176      2     11 0.982575   
#> Residuals 12                                           
#> ---
#> Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1
eta_squared(m)
#> Parameter | Eta2 (partial) |       90% CI
#> -----------------------------------------
#> block     |           0.44 | [0.00, 0.55]
#> N         |           0.61 | [0.20, 0.77]
#> P         |           0.07 | [0.00, 0.29]
#> K         |           0.39 | [0.00, 0.59]
#> N:P       |           0.11 | [0.00, 0.36]
#> N:K       |           0.17 | [0.00, 0.45]
#> P:K       |       3.19e-03 | [0.00, 0.00]
model_parameters(m)
#> Parameter |   Pillai | df |    F |     p
#> ----------------------------------------
#> block     |     0.88 |  5 | 1.90 | 0.096
#> N         |     0.61 |  1 | 8.52 | 0.006
#> P         |     0.07 |  1 | 0.39 | 0.686
#> K         |     0.39 |  1 | 3.49 | 0.067
#> N:P       |     0.11 |  1 | 0.65 | 0.541
#> N:K       |     0.17 |  1 | 1.16 | 0.350
#> P:K       | 3.19e-03 |  1 | 0.02 | 0.983
#> Residuals |          | 12 |      |

# maov
summary(aov(m))
#>  Response yield :
#>             Df Sum Sq Mean Sq F value   Pr(>F)   
#> block        5 343.29  68.659  4.4467 0.015939 * 
#> N            1 189.28 189.282 12.2587 0.004372 **
#> P            1   8.40   8.402  0.5441 0.474904   
#> K            1  95.20  95.202  6.1657 0.028795 * 
#> N:P          1  21.28  21.282  1.3783 0.263165   
#> N:K          1  33.13  33.135  2.1460 0.168648   
#> P:K          1   0.48   0.482  0.0312 0.862752   
#> Residuals   12 185.29  15.441                    
#> ---
#> Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1
#> 
#>  Response foo :
#>             Df  Sum Sq Mean Sq F value Pr(>F)
#> block        5  3.9505 0.79010  0.6122 0.6929
#> N            1  1.4921 1.49208  1.1561 0.3034
#> P            1  0.0794 0.07944  0.0615 0.8083
#> K            1  0.0612 0.06124  0.0474 0.8312
#> N:P          1  0.0730 0.07304  0.0566 0.8160
#> N:K          1  0.0021 0.00215  0.0017 0.9681
#> P:K          1  0.0003 0.00030  0.0002 0.9881
#> Residuals   12 15.4878 1.29065
eta_squared(aov(m))
#> Response | Parameter | Eta2 (partial) |       90% CI
#> ----------------------------------------------------
#> foo      |     block |           0.20 | [0.00, 0.35]
#> foo      |         N |           0.09 | [0.00, 0.38]
#> foo      |         P |       5.10e-03 | [0.00, 0.19]
#> foo      |         K |       3.94e-03 | [0.00, 0.17]
#> foo      |       N:P |       4.69e-03 | [0.00, 0.18]
#> foo      |       N:K |       1.39e-04 | [0.00, 0.00]
#> foo      |       P:K |       1.92e-05 | [0.00, 0.00]
#> yield    |     block |           0.65 | [0.16, 0.78]
#> yield    |         N |           0.51 | [0.15, 0.71]
#> yield    |         P |           0.04 | [0.00, 0.32]
#> yield    |         K |           0.34 | [0.03, 0.60]
#> yield    |       N:P |           0.10 | [0.00, 0.40]
#> yield    |       N:K |           0.15 | [0.00, 0.45]
#> yield    |       P:K |       2.59e-03 | [0.00, 0.15]
model_parameters(aov(m))
#> # yield response
#> 
#> Parameter | Sum_Squares | df | Mean_Square |     F |     p
#> ----------------------------------------------------------
#> block     |      343.29 |  5 |       68.66 |  4.45 | 0.016
#> N         |      189.28 |  1 |      189.28 | 12.26 | 0.004
#> P         |        8.40 |  1 |        8.40 |  0.54 | 0.475
#> K         |       95.20 |  1 |       95.20 |  6.17 | 0.029
#> N:P       |       21.28 |  1 |       21.28 |  1.38 | 0.263
#> N:K       |       33.13 |  1 |       33.13 |  2.15 | 0.169
#> P:K       |        0.48 |  1 |        0.48 |  0.03 | 0.863
#> Residuals |      185.29 | 12 |       15.44 |       |      
#> 
#> # foo response
#> 
#> Parameter | Sum_Squares | df | Mean_Square |        F |     p
#> -------------------------------------------------------------
#> block     |        3.95 |  5 |        0.79 |     0.61 | 0.693
#> N         |        1.49 |  1 |        1.49 |     1.16 | 0.303
#> P         |        0.08 |  1 |        0.08 |     0.06 | 0.808
#> K         |        0.06 |  1 |        0.06 |     0.05 | 0.831
#> N:P       |        0.07 |  1 |        0.07 |     0.06 | 0.816
#> N:K       |    2.15e-03 |  1 |    2.15e-03 | 1.67e-03 | 0.968
#> P:K       |    2.98e-04 |  1 |    2.98e-04 | 2.31e-04 | 0.988
#> Residuals |       15.49 | 12 |        1.29 |          |
```

<sup>Created on 2020-11-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by IndrajeetPatil** [comment]
I have no idea why the tests are failing selectively on `ubuntu` even with `tolerance = 0.001`.

File: DESCRIPTION [hunk 4]
```diff
@@ -71,7 +71,7 @@ Suggests:
     cplm,
     dplyr,
     DRR,
-    effectsize (>= 0.4.0),
+    effectsize (>= 0.4.0.001),
```
Comment (strengejacke) [inline DESCRIPTION:74]: I would rather leave this as is (i.e. >= 0.4.0), and then check the package version in the test file (i.e. run tests conditionally if package version of effectsize is > 0.4.0), else there might be troubles when updating / loading the package, using _easystats_ etc.

**Review by strengejacke — CHANGES_REQUESTED** [review]

File: DESCRIPTION [hunk 5]
```diff
@@ -130,3 +130,7 @@ Encoding: UTF-8
 Language: en-US
 RoxygenNote: 7.1.1
 VignetteBuilder: knitr
+Remotes:
```
Comment (strengejacke) [inline DESCRIPTION:133]: I don't think there is an official "Remotes" tag in R / CRAN, or? So I would rather remove this

**Review by strengejacke — APPROVED** [review]

**Comment by IndrajeetPatil** [comment]
Sorry, just came back to the computer. I see you have addressed your own comments. Thanks!

Hmm, I am not sure why the tests are now failing on all OS.

**Comment by strengejacke** [comment]
I don't know, either... Maybe we should remove or disable these tests for now?

**Comment by IndrajeetPatil** [comment]
Yeah, I agree. Let's skip the strict tests for now and we can revisit them later.

**Pull request merged**